src/media/js/error_messages.js

```diff
@@ -0,0 +1,26 @@
+/*
+    Error codes and their associated messages.
+*/
+define('error_messages', ['core/l10n'], function(l10n) {
+    'use strict';
+    var gettext = l10n.gettext;
+
+    return {
```

Comment:
I like the personal touch of specific error messages but would be also OK with 'item_missing': 'We had trouble loading the details of this item...'